great_expectations/expectations/core/expect_compound_columns_to_be_unique.py

```diff
@@ -109,3 +109,15 @@ def _prescriptive_renderer(
                 }
             )
         ]
+
+    @classmethod
+    @renderer(renderer_type="renderer.diagnostic.observed_value")
+    def _diagnostic_observed_value_renderer(
+        cls,
+        configuration=None,
+        result=None,
+        language=None,
+        runtime_configuration=None,
+        **kwargs,
+    ):
+        pass
```

Comment:
Shouldn't this be the percent passing? This should be basically identical to non-multicolumn map, and available on the parent.

Reply:
Deleted this section (parent handles).